packages/foam-vscode/static/graphs/default/graph.js

```diff
@@ -31,6 +34,10 @@ const defaultStyle = {
   background: getStyle(`--vscode-panel-background`) ?? styleFallback.background,
   fontSize:
     parseInt(getStyle(`--vscode-font-size`) ?? styleFallback.fontSize) - 2,
+  lineColor:
+    getStyle('--vscode-list-deemphasizedForeground') ?? styleFallback.lineColor,
```

Comment:
before this PR we were computing the line color with `d3.hsl(style.node.note).darker(2);`. 
Can we keep the same behavior in the default style? I am not sure that `--vscode-list-deemphasizedForeground` is a good enough approximation across themes

Reply:
Yes - did I understand right that, set it to `--vscode-editor-foreground` by default instead?  I'll fix that right away too.

Reply:
originally I thought we should by default have `lineColor` be based off of the value of `node.note`, not `--vscode-editor-foreground`, so that if a user only changes `node.note`, it automatically changes the line color accordingly. 
The user can still manually override `lineColor` of course. 
Still debating whether that's a good idea, both because I think it makes things a little bit more complicated (nothing too crazy) and because maybe the user wouldn't expect that change (which, again, they could still reverse by providing a `lineColor` value. What do you think?

Instead of `.darker(2)` we can use the same transparency trick you have used everywhere else (e.g. `.opacity(.8)`) to make this work across light and dark themes.

Reply:
Sorry for the delay - day job got in the way. I agree that this is a much better approach - allowing the user to basically get away with only having to set background color and node color and most settings will fall into place. I've made the change and pushed it - let me know if I misunderstood anything :)